doc/Navigation/HowTo-DisplayItem.md

```diff
@@ -6,138 +6,165 @@ This topic walks through how to use Navigation to display the details of an item
 
-```csharp
-new ViewMap<SecondPage, SecondViewModel>(Data: new DataMap<Widget>())
-```
+    ```csharp
+    new ViewMap<SecondPage, SecondViewModel>(Data: new DataMap<Widget>())
+    ```
 
-- Create a widget inside the main view model, and pass it as `data` into the navigation method
+### 2. Pass data when navigating
+- Create a `Widget` inside the `GoToSecondPage` method in `MainViewModel.cs`, and pass it as `data` into the navigation method.
        
-```csharp
-public async Task GoToSecondPage()
-{
-	var widget = new Widget("CrazySpinner", 34.0);
-
-	await _navigator.NavigateViewModelAsync<SecondViewModel>(this, data: widget);
-}
-```
-
-- Update the second view model constructor to accept a widget
-
-```csharp
-public class SecondViewModel
-{
-    public string Name { get; }
+    ```csharp
+    public async Task GoToSecondPage()
+    {
+    	var widget = new Widget("CrazySpinner", 34.0);
     
-    public SecondViewModel(Widget widget)
-	{
-        Name = widget.Name; 
-	}
-}
-```
-
-- Add a TextBlock to second page XAML
-
-```xml
-<TextBlock HorizontalAlignment="Center"
-                   VerticalAlignment="Center"><Run Text="Widget Name:" /><Run Text="{Binding Name}" /></TextBlock>
-```          
-- Show (screenshot) app on second page showing widget name            
-            
-- Because there's a mapping between the Widget and the SecondViewModel, can just navigate to show data. Change navigate method to NavigateDataAsync
-
-```csharp
-await _navigator.NavigateDataAsync(this, data: widget);
-```
-
-
-- Add a `Widgets` property to your main view model, similar to the following:
-
-```csharp
-public Widget[] Widgets { get; } = new[]
-{
-    new Widget("NormalSpinner", 5.0),
-    new Widget("HeavySpinner",50.0)
-};
-```
-
-- Update main page XAML to display widgets
-
-```xml
-<ListView ItemsSource="{Binding Widgets}"
+    	await _navigator.NavigateViewModelAsync<SecondViewModel>(this, data: widget);
+    }
+    ```
+
+### 3. Receiving navigation data
+- Update the `SecondViewModel` constructor to accept a `Widget` parameter.
+
+    ```csharp
+    public class SecondViewModel
+    {
+        public string Name { get; }
+        
+        public SecondViewModel(Widget widget)
+    	{
+            Name = widget.Name; 
+    	}
+    }
+    ```
+
+- Add a `TextBlock` to `SecondPage.xaml`
+    ```xml
+    <TextBlock  HorizontalAlignment="Center"
+                VerticalAlignment="Center">
+        <Run Text="Widget Name:" /><Run Text="{Binding Name}" />
+    </TextBlock>
+    ```          
+     
+### 4. Navigating with data            
+- Because there's a mapping between the `Widget` and the `SecondViewModel` (in the `ViewMap` defined in `App.xaml.host.cs`), an alternative way to navigate is by calling the `NavigateDataAsync` and specifying the data object to pass in the navigation request. The type of the data object will be used to revolve which route to navigate to
```

Comment:
```suggestion
- Because there's a mapping between the `Widget` and the `SecondViewModel` (in the `ViewMap` defined in `App.xaml.host.cs`), an alternative way to navigate is by calling the `NavigateDataAsync` and specifying the data object to pass in the navigation request. The type of the data object will be used to revolve which route to navigate to:
```